docker-compose.yaml

```diff
@@ -0,0 +1,72 @@
+version: '3'
+
+# This docker-compose is for developer convenience, not for running in production.
+
+services:
+
+  namenode:
```

Comment:
Could you include a description of each node in the README? This would be helpful for those of us who aren't very familiar with Yarn.

Reply:
Done